packages/electron-publish/src/gitHubPublisher.ts

```diff
@@ -215,6 +215,19 @@ export class GitHubPublisher extends HttpPublisher {
       })
   }
 
+  private doesErrorMeanAlreadyExists(e: any) {
+    let doesErrorMeanAlreadyExists = true
+    doesErrorMeanAlreadyExists = doesErrorMeanAlreadyExists && e.statusCode === 422
+    if (e.description !== null) {
```

Comment:
this actually fails if description is undefined, so I'll push another fixup that resolves that situation as well

Reply:
fixed